### PR TITLE
Update images.mdx

### PR DESCRIPTION
**Pull request by K1ri86**
#### What kind of changes does this PR include?

- Updated content
- Translated content

#### Description

- Translation into Spanish of the latest modifications

### CODE REVIEW

File: src/content/docs/es/guides/images.mdx [hunk 1]
```diff
@@ -654,7 +654,27 @@ Si estabas utilizando la integración de imágenes en Astro v2.x, completa los s
     })
     ```
 
-2. Migra cualquier componente `<Image />` existente.
+2. Actualizar tipos (si es necesario).
```
Comment (Waxer59) [inline src/content/docs/es/guides/images.mdx:657]:
```suggestion
2. Actualiza los tipos (si es necesario).
```

File: src/content/docs/es/guides/images.mdx [hunk 2]
```diff
@@ -654,7 +654,27 @@ Si estabas utilizando la integración de imágenes en Astro v2.x, completa los s
...
+
+		Si tenías tipos especiales configurados para `@astrojs/image` en `src/env.d.ts`, es posible que necesites cambiarlos de nuevo a los tipos predeterminados de Astro si tu actualización a la versión 3 no completó este paso por ti.
```
Comment (Waxer59) [inline src/content/docs/es/guides/images.mdx:659]:
```suggestion
		Si tenías tipos especiales configurados para `@astrojs/image` en `src/env.d.ts`, es posible que necesites cambiarlos de nuevo a los tipos predeterminados de Astro si la actualización a la versión 3 no completó este paso por ti.
```

File: src/content/docs/es/guides/images.mdx [hunk 3]
```diff
@@ -654,7 +654,27 @@ Si estabas utilizando la integración de imágenes en Astro v2.x, completa los s
...
+		```json title="tsconfig.json" del={3} ins={4}
+		{
+			"compilerOptions": {
+			  "types": ["@astrojs/image/client"]
+			  "types": ["astro/client"]
+			}
+		}
+		```
+
+3. Migre cualquier componente `<Imagen />` existente.
```
Comment (Waxer59) [inline src/content/docs/es/guides/images.mdx:677]:
```suggestion
3. Migra cualquier componente `<Imagen />` existente.
```

**Review by Waxer59 — COMMENTED** [review]
Hola!, gracias por la PR 🤠!, te dejo unas sugerencias

File: src/content/docs/es/guides/images.mdx [hunk 4]
```diff
@@ -654,7 +654,27 @@ Si estabas utilizando la integración de imágenes en Astro v2.x, completa los s
     })
     ```
 
-2. Migra cualquier componente `<Image />` existente.
+2. Actualizar los tipos (si es necesario).
```
Comment (Waxer59) [inline src/content/docs/es/guides/images.mdx:657]:
```suggestion
2. Actualiza los tipos (si es necesario).
```

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @K1ri86, welcome to Team Docs! 🙌

**Pull request merged**